templates/aspnet-s2i-template.json

```diff
@@ -280,7 +277,7 @@
         {
             "name": "ASP_UPSTREAM_IMAGE_TAG",
             "description": "Base OpenShift ASP S2I Image Tag",
-            "value": "latest",
+            "value": "1.0",
```

Comment:
In this case, I don't think it matters. The template defines an image stream, called `dotnet` which uses the 1.0 image. So dotnet:latest will refer to latest 1.0. That's my take on it anyhow.